v2/lints/etsi/lint_qcstatem_qclimitvalue_valid.go

```diff
@@ -53,38 +45,36 @@ func isOnlyLetters(s string) bool {
 }
 
 func (l *qcStatemQcLimitValueValid) Execute(c *x509.Certificate) *lint.LintResult {
+	errString := util.ErrorStringBuilder{Delimiter: "; "}
+
+	if len(c.QCStatements.ParsedStatements.Limit) != 1 {
```

Comment:
When the QCStatement parsing was landed to ZCrypto, I couldn't find any language in the relevant specifications to assert that one and only one instance of a given QCStatement could be present. The `UNIQUE` in the `QC-STATEMENT` information object only means that there will only be one unique statement within the information object set, not that there can be only one unique instantiation. 

X.681 reiterates this in the introduction and explicitly in 3.4.8

Reply:
The motivation for this was to avoid ambiguity rather than to address an ASN.1 constraint. If more than one QcEuLimitValue exist an application that uses the certificate would have difficulties to decide which one to use. Some implementations would probably use the first occurence.

If that is too strict or out of scope it is fine to delete this.